observability/monitoring/common-monitoring-configuration-scenarios.adoc

```diff
@@ -0,0 +1,53 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="common-monitoring-configuration-scenarios"]
+= Common monitoring configuration scenarios
+include::_attributes/common-attributes.adoc[]
+:context: common-monitoring-configuration-scenarios
+
+toc::[]
+
+The default platform monitoring stack is available for use after installing {product-title}.
+The in-cluster stack includes the core platform Prometheus instance that collects metrics from your cluster and the core Alertmanager instance that routes alerts.
+
+In addition to core platform monitoring, you can also optionally xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[enable monitoring for user-defined projects] for user workload monitoring.
+After doing so, users can then monitor their own services and workloads without the need for an additional monitoring solution.
+
+Depending on which users will use the monitoring stack and for what purposes, as a cluster administrator, you can configure monitoring components to suit the needs of different users in various scenarios.
+
+[id="core-platform-monitoring-first-steps_{context}"]
+== Core platform monitoring: First steps
+
+After {product-title} is installed, cluster administrators typically perform certain basic activities to configure core platform monitoring to suit their needs.
+These activities include configuring options for Prometheus and Alertmanager, setting up short-term and long-term storage, and adding service monitors.
+When you have completed these configuration activities, users can query and view collected metrics and receive alerts.
+
+* xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#granting-users-permission-to-monitor-user-defined-projects_enabling-monitoring-for-user-defined-projects[Assign monitoring cluster roles] to any non-administrator users that need to access certain monitoring features.
+The `monitoring-edit` cluster role permits users to create alerts and to create, modify, and delete `ServiceMonitor` and `PodMonitor` resources to scrape metrics from services and pods.
```

Comment:
```suggestion
The `monitoring-edit` cluster role permits users to create alerts and to create, modify, and delete `ServiceMonitor`, `prometheusrules`  and `PodMonitor` resources to scrape metrics from services and pods.
```
see from
```
$ oc get clusterrole monitoring-edit -oyaml
...
rules:
- apiGroups:
  - monitoring.coreos.com
  resources:
  - servicemonitors
  - podmonitors
  - prometheusrules
  verbs:
  - '*'
```

Reply:
Please note https://docs.openshift.com/container-platform/4.16/observability/monitoring/configuring-the-monitoring-stack.html#support-considerations_configuring-the-monitoring-stack

Perhaps we should add a note that this targets UWM specifically and not the platform part?

Reply:
Changed to say:

`The `monitoring-edit` cluster role permits users to create alerts and to create, modify, and delete `ServiceMonitor`, `PrometheusRules` and `PodMonitor` resources to scrape metrics from services and pods for user-defined projects only.`